Sources/BraintreeDropIn/Models/BTDropInResult.m

```diff
@@ -42,7 +42,13 @@ @implementation BTDropInResult
 - (instancetype)init {
     self = [super init];
     if (self) {
-        _deviceData = [PayPalDataCollectorClass collectPayPalDeviceData];
+        // If we are testing we want to set PayPalDataCollectorClass to the class passed in `setPayPalDataCollectorClass`
+        if (PayPalDataCollectorClass != NSClassFromString(PayPalDataCollectorClassString)) {
+            _deviceData = [PayPalDataCollectorClass collectPayPalDeviceData];
+            return self;
+        }
+        // Otherwise we should use `PPDataCollector` to collect device data
+        _deviceData = [PPDataCollector collectPayPalDeviceData];
```

Comment:
This is wild. So the test was injecting a definition for `PayPalDataCollectorClass`? So glad we're moving to Swift lol.

Reply:
Yeah, the test was setting a class, but the actual implementation never was - so tests were passing with the mock class being injected and the live code was returning nil for the past year 😢 Swift will certainly help us catch these cases sooner!